### PR TITLE
Do not use fork of openzeppelin anymore

### PR DESCRIPTION
**Pull request by axic**
Depends on https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1498

### CODE REVIEW

**Comment by chriseth** [comment]
Fails.

**Comment by nventuro** [comment]
Hey @axic, I noticed your build is failing because of the breaking changes introduced in 0.5.0 (e.g. the data location for certain types now being mandatory). I was actually tracking that in [this OpenZeppelin PR](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1068), and we even have an issue for this whole thing: https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1211. 

What do you think about having a branch in OpenZeppelin that compiles using nightly builds, instead of you having to maintain your own fork? That way we could give you a hand maintaining, making sure the branch gets updated often to reflect the latest changes, and it'd also make it easier for us to upgrade to the latest version once its released.

**Comment by chriseth** [comment]
@nventuro that would be wonderful! Do you already have such a branch?

**Comment by nventuro** [comment]
I just created the [`solc-nightly`](https://github.com/OpenZeppelin/openzeppelin-solidity/tree/solc-nightly) branch, from `master`, and set the CI to only use the nightly compiler. To bring that branch up to date with 0.5.0, we should open PRs against it: we already have a couple from a while back (https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1068 and https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1144).

**Comment by chriseth** [comment]
Will you make the updates? You know about https://github.com/axic/openzeppelin-solidity/tree/solidity-050 do you?

**Comment by axic** [comment]
> You know about https://github.com/axic/openzeppelin-solidity/tree/solidity-050 do you?

The problem is we haven't merged upstream changes to it for 3 months. We could do that, but I think merging these 2 new PRs will be faster.

**Comment by chriseth** [comment]
No reason to keep this open.

**Comment by axic** [comment]
Zeppelin has released their 0.5.0-compatible version: https://github.com/OpenZeppelin/openzeppelin-solidity/releases/tag/v2.1.1 Updating it here.

**Comment by axic** [comment]
This works now.

File: test/externalTests.sh [hunk 1]
```diff
@@ -59,10 +59,13 @@ function test_truffle
       for d in node_modules node_modules/truffle/node_modules
       do
       (
-        cd $d
-        rm -rf solc
-        git clone --depth 1 -b v0.5.0 https://github.com/ethereum/solc-js.git solc
-        cp "$SOLJSON" solc/
+        if [ -d "$cd" ]
```
Comment (chriseth) [inline test/externalTests.sh]: `$cd`?

Comment (axic) [reply test/externalTests.sh]: Damn, I'm tired, was supposed to be `$d`.

**Review by chriseth — APPROVED** [review]

**Pull request merged**